Sources/Fog/View/FogView.swift

```diff
@@ -62,17 +65,23 @@ final class FogView {
         searchResult: FogView_TxOutSearchResult,
         accountKey: AccountKey
     ) -> Result<LedgerTxOut, ConnectionError> {
-        FogViewUtils.decryptTxOutRecord(
+        logger.info("")
+        return FogViewUtils.decryptTxOutRecord(
             ciphertext: searchResult.ciphertext,
             accountKey: accountKey
         ).mapError { error in
             switch error {
             case .invalidInput(let reason):
-                print("Warning: could not decrypt TxOut returned from Fog View, base64 " +
-                    "ciphertext: \(searchResult.ciphertext.base64EncodedString()), " +
-                    "error: \(error)")
+                logger.info(
+                    "Warning: could not decrypt TxOut returned from Fog View, base64 " +
+                        "ciphertext: \(searchResult.ciphertext.base64EncodedString()), " +
+                        "error: \(redacting: error)")
```

Comment:
I believe `error` will have already been redacted, since it's not a LibMobileCoin error. If not we should make sure to redact it at the place where the error is created.